frontend/catalyst/jit.py

```diff
@@ -535,6 +537,10 @@ def aot_compile(self):
                 self.compiled_function, self.user_sig, self.out_treedef, self.workspace
             )
 
+        # After compilation is successfully done, remove the transform primitives from jaxpr
+        if self.jaxpr is not None:
+            erase_transform_primitives_from_jaxpr(self.jaxpr)
```

Comment:
Functionally no reason, as we never run the jaxpr itself. However there are some lit tests checking for jaxpr and they don't have this. I guess if we are ok with the design of adding the transform_named_sequence primitive to all tracings then it would make sense to modify these tests instead of clean up here.

Reply:
I have removed this erasing. The transform primitives will be kept in the jaxpr.